content/en/serverless/step_functions/installation.md

```diff
@@ -30,139 +30,63 @@ Ensure that the [AWS Step Functions integration][9] is installed.
-[4]: logs/guide/forwarder/?tab=cloudformation#installation
-[5]: https://github.com/DataDog/datadog-ci/blob/master/src/commands/stepfunctions/README.md
-{{% /tab %}}
-{{% tab "Custom" %}}
-
 1. Enable all logging for your Step Function. In your AWS console, open your state machine. Click *Edit* and find the Logging section. There, set *Log level* to `ALL` and enable the *Include execution data* checkbox.
    {{< img src="serverless/step_functions/aws_log.png" alt="AWS UI, Logging section, showing log level set to ALL." style="width:100%;" >}}
 
-2. Ensure you have deployed the [Datadog Lambda Forwarder][1], a Lambda function that ships logs from AWS to Datadog, and that you are using v3.121.0+. You may need to [update your Forwarder][2]. When deploying the Forwarder on v3.121.0+, you can also set the `DdStepFunctionsTraceEnabled` parameter in CloudFormation to enable tracing for all your Step Functions at the Forwarder-level.
-
-   Take note of your Forwarder's ARN.
-
-3. Subscribe CloudWatch logs to the Datadog Lambda Forwarder. To do this, you have two options:
-   - **Datadog-AWS integration** (recommended)
-     1. Ensure that you have set up the [Datadog-AWS integration][4].
-     2. In Datadog, open the [AWS integration tile][5], and view the *Configuration* tab.
-     3. On the left, select the AWS account where your Step Function is running. Open the *Log Collection* tab.
-     4. In the *Log Autosubscription* section, under *Autosubscribe Forwarder Lambda Functions*, enter the ARN of your Datadog Lambda Forwarder, as noted previously. Click *Add*.
-     5. Toggle on *Step Functions CloudWatch Logs*. Changes take 15 minutes to take effect.
-
-     **Note**: Log Autosubscription requires your Lambda Forwarder and Step Function to be in the same region.
-
-   - **Manual**
-     1. Ensure that your log group name has the prefix `/aws/vendedlogs/states`. 
-     2. Open your AWS console and go to your Datadog Lambda Forwarder. In the *Function overview* section, click on *Add trigger*.
-     3. Under *Add trigger*, in the *Trigger configuration* section, use the *Select a source* dropdown to select `CloudWatch Logs`.
-     4. Under *Log group*, select the log group for your state machine. For example, `/aws/vendedlogs/states/my-state-machine`.
-     5. Enter a filter name. You can choose to name it "empty filter" and leave the *Filter pattern* box blank.
-
-<div class="alert alert-warning"> If you are using a different instrumentation method, such as Serverless Framework or datadog-ci, enabling autosubscription may create duplicated logs. To avoid this behavior, choose only one configuration method.</div>
+2. Ensure you have deployed the [Datadog Lambda Forwarder][11], and that you are using v3.121.0+. You might need to [update your Forwarder][12]. As an alternative, you may also use [Amazon Data Firehose][16].
+
+3. Subscribe CloudWatch logs to the Datadog Lambda Forwarder. 
+   {{< tabs >}}
+   {{% tab "Automatic" %}}
```

Comment:
The automatic/manual steps need to become 4 different tabs

"Serverless-Plugin","Dataodg-CI", "Datadog UI" "Manually on AWS"

The Serverless Plugin, Datadog-Ci actually do steps including log autosubscription AND enable tracing via the envrionment variable

Reply:
@sumedham,Here's some context based on my understanding
-The setup via CI/Plugin does not work with customized log groups. It is only functional when the Step Function (SF) log group name starts with /aws/vendedlogs/states [(source code link)](https://github.com/DataDog/serverless-plugin-datadog/blob/11fe0b87a65a3847adcdd9a2ab3de699867ee7f5/src/forwarder.ts#L122).
- Once the subscription is successfully set up for a deployed SF, there is no need to repeat the process for future deployments.
- Low Adoption of CI/Plugin for SF Deployment: As you know, only a small number of customers currently use CI/Plugin to deploy Step Functions.

Given the complexity of the existing documentation, I believe it may not be worth explicitly adding separate tabs for CI and Plugin in this section. I think we basically want to customers to use our product no matter which deployment method they are using of. However, we can include a note in this section, mentioning that:
- If customers are using /aws/vendedlogs/states as log group name prefixes and deploying Step Functions using CI/Plugin, they can leverage CI/Plugin to set up this subscription.
- Provide links to the public documentation for both CI and Plugin to guide users.
- 
WDYT?